app/templates/admin/createSingleEvent.html

```diff
@@ -244,18 +244,22 @@ <h1 id="title">{{page_title}}</h1>
           <div class="col-md-6 offset-md-3">
             <div class="text-center">
               {% if not isPastEvent  %}
-                {% if userHasRSVPed %}
-                  <button type="submit" class="btn btn-danger"  formaction="/rsvpRemove">Remove RSVP</button>
-                {% else %}
-                  <button type="submit" id="rsvpEvent" class="btn btn-primary" formaction="/rsvpForEvent" {{'disabled' if isPastEvent}}>RSVP</button>
-                {% endif %}
-              {% endif %}
-
+                  {% if userHasRSVPed %}
+                    <button type="submit" class="btn btn-danger"  formaction="/rsvpRemove">Remove RSVP</button>
+                  {% else %}
+                     {% if request.path[-6:] != "create"%}
```

Comment:
This is still wrong. There is already a variable in the template for this logic.